INSTALL.rst

```diff
@@ -100,10 +99,9 @@ Go to the section `Check that installation was successful`_.
 .. [1] See the `conda glossary`_ for the differences between Anaconda and Miniconda, and the definitions of the terms ‘channel’ and ‘environment’ here.
 .. [2] The ‘$’ character at the start of these lines indicates that the command text should be entered in the terminal or prompt, depending on the operating system.
        Do not retype the ‘$’ character itself.
-.. [3] "strict" channel priority ensures that conda will select the latest version of message_ix from the "noarch" channel, and ignore much older (v1.0.0) packages in OS-specific channels.
-       See `managing channels`_ of the Anaconda documentation for more information.
-.. [4] Notice that conda uses the hyphen (‘-’) in package names, different from the underscore (‘_’) used in Python when importing the package.
-
+.. [3] Notice that conda uses the hyphen (‘-’) in package names, different from the underscore (‘_’) used in Python when importing the package.
+       Specifying a version >1 is currently required on macOS.
+       On Windows, it is optional; one can also use conda install message-ix.
```

Comment:
Please use:
- Double-backticks around `conda install message-ix` so it is formatted in a code/fixed width font.
- Asterisks (italics) on `*required*` and `*optional*`.

Reply:
Thank you for your review!